src/hapi/pipelines/app/pipelines.py

```diff
@@ -169,6 +206,7 @@ def output(self):
         self.org.populate()
         self.org_type.populate()
         self.sector.populate()
+        self.currency.populate()
```

Comment:
I know you explained this and I should already understand but: why are the other tables not configurable scrapers?

Reply:
currency and wfp_commodity do not map to admins. food price datasets are found by reading a file in a resource that lists them all rather than specifying explicitly in YAML. wfp_market might work in a configurable scraper. I found it easier to keep it like the others and also I could customise the logging.